backend/src/main/java/com/empire/World.java

```diff
@@ -35,7 +35,11 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-class World {
+interface GoodwillProvider {
```

Comment:
Instituting this kind of change, along with privatizing every class member we can, will be a huge help in every way I can think of for refactoring/new development.  I wonder if it's just going to result in World implementing 50 interfaces by the end and whether that is ideal.  Regardless, it should be easier to pivot to any kind of pattern afterwards with all these functions isolated in interfaces.

Reply:
Yeah. My thinking is that World probably will start out implementing a ton of interfaces, but as it gets more broken up some of those will find new homes, and some could feasibly be multiple function interfaces (but then you lose the lambdas) - basically, I'd like to try it and see how it goes.